extensions/blocks/business-hours/business-hours.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * Business Hours Block.
+ *
+ * @since 7.1.0
+ *
+ * @package Jetpack
+ */
+
+register_block_type(
+	'jetpack/business-hours',
+	array( 'render_callback' => 'jetpack_business_hours_render' )
+);
+
+/**
+ * Business Hours Block dynamic rending of the glock.
```

Comment:
Not introduced here, but: typo galore :sweat_smile:

Should be `dynamic rendering of the block` I guess :grimacing:

Reply:
Fixed in ba0c2c1d51b4807adfdf561f1249ad182d921762